__tests__/html/scrollToEndButton.tabOrder.html

```diff
@@ -51,9 +51,7 @@
 
         await pageConditions.allImagesLoaded();
 
-        await host.sendTab();
-
-        // After pressing TAB from transcript, it should focus on the "New messages" button.
+        // When "New message" appears then focus should be automatically move to "New messages" button.
```

Comment:
Please revert the change and add a dedicated test covering changes made in the PR

Reply:
We cannot move the focus when "New messages" button appear. This is violating accessibility compliance.

This is because the message come asynchronously and moving the focus will interrupt the screen reader.

Reply:
Fixed in latest commit